Source/WTF/wtf/HashFunctions.h

```diff
@@ -62,7 +62,7 @@ namespace WTF {
     }
 
     // Thomas Wang's 32 Bit Mix Function: http://www.cris.com/~Ttwang/tech/inthash.htm
-    inline unsigned intHash(uint32_t key) 
+    inline unsigned intHash(uint32_t key)
```

Comment:
All the changes to this file are just trimming white space. I can't think of a compelling reason to do that and such changes should not be combined with substantive changes in the same pull request.